package.json

```diff
@@ -54,7 +54,7 @@
     "@artsy/express-reloadable": "1.4.0",
     "@artsy/palette": "2.23.2",
     "@artsy/passport": "1.1.0",
-    "@artsy/reaction": "9.3.4",
+    "@artsy/reaction": "9.3.8",
```

Comment:
Shouldn't need to bump reaction but rather merge the automatic renovate pr

Reply:
Ah, I closed that one (since doing it manually here).

What if renovate is automatically merged and this separate one is forgotten? That could be...bad. I think it makes better sense/safer for a single PR to contain the Reaction bump + any accompanying code change that's needed.

For regular bumps though, I like the Renovate thing!

Reply:
This is fine for now but in the future its best to stay within renovate (and then rebase off master) since we don't want to get in a position where there are two competing updating mechanisms. The process is pretty instant, and if its a patch or minor release things are pretty safe.

Reply:
Not sure if I agree honestly, if there's a breaking Reaction change, wouldn't having that automatically update in Force potentially lead to....issues.

Say someone merges a big Reaction PR, and then everyone is out to lunch or whatever. In the intervening time Force is thus broken b/c that was automatically merged. Then, a separate developer goes to deploy Force during that time...and 🔥 .

I feel like the auto updates are fine for the smaller/no-op stuff, but the more manual approach is preferred for bigger ones? Or is the move there to make sure the bigger PR's on Reaction are sent out with a 'major' (or 'minor') release, and then Renovate can _only_ auto-update 'patch' ones?

Reply:
It isn't automatically merged -- A PR is opened and then whoever triggered the change is (typically) responsible for merging it.